master/buildbot/scripts/base.py

```diff
@@ -69,7 +69,11 @@ def checkPidFile(pidfile):
                 raise OSError("Can't check status of PID {} from pidfile {}: {}".format(
                     pid, pidfile, why))
         else:
-            raise BusyError("'{}' exists - is this master still running?".format(pidfile))
+            # No exception if pid equal ppid, fixed it.
+            if pid == os.getppid():
```

Comment:
I think this deserves a bit more comments.
Anyway I am not sure if this will not bite us later. Since this is a docker specific issue, I wonder if we'd rather delete the pidfile in the docker entrypoint shell script.

Reply:
Agree, new PR at #3606 , I'm close this.